cms/envs/production.py

```diff
@@ -697,3 +697,14 @@ def get_env_setting(setting):
 # .. for now it wil impact country listing in auth flow and user profile.
 # .. eg ['US', 'CA']
 DISABLED_COUNTRIES = ENV_TOKENS.get('DISABLED_COUNTRIES', [])
+
+# .. setting_name: LIBRARY_ENABLED_BLOCKS
+# .. setting_default: ['problem', 'video', 'html', 'drag-and-drop-v2']
+# .. setting_description: List of block types that are enabled to be created/used
+# .. in libraries. Both basic blocks and advanced blocks can be included.
+# .. In the future, advanced block configuration will be handled individually per library.
+# .. eg ['problem', 'video', 'drag-and-drop-v2]
```

Comment:
```suggestion
```
Settings only need to be annotated once. This is already annotated in common.py.

Reply:
Deleted here https://github.com/openedx/edx-platform/pull/36292/commits/057492327404071cecc1d4d9178efacb7cc79636